xgboostserver/rockcraft.yaml

```diff
@@ -0,0 +1,35 @@
+name: xgboostserver
+base: ubuntu:20.04
+version: "1.16.0"  # Should match source-tag below
+summary: Seldon XGBoost Server
+description: |
+  The model server for XGBoost models
+license: Apache-2.0
+
+platforms:
+    amd64:
+
+parts:
+    xgboostserver:
+        plugin: python
+        source: https://github.com/SeldonIO/seldon-core
+        source-type: git
+        source-tag: "v1.16.0"  # Should match 'version' above. version.txt is the same, just without the 'v' prefix
+        python-requirements:
+          - servers/xgboostserver/xgboostserver/requirements.txt
+        python-packages:
+          - seldon_core
+        override-build: |
+          craftctl default
+          install -D -m755 servers/xgboostserver/xgboostserver/XGBoostServer.py ${CRAFT_PART_INSTALL}/opt/XGBoostServer.py
+        organize:
+          opt/XGBoostServer.py: usr/bin/XGBoostServer.py
+        stage-packages:
+          - python3-venv
+
+services:
+  xgboostserver:
+    override: merge
+    command: /bin/python3 /usr/bin/XGBoostServer.py
```

Comment:
Providing a full path here because of a [rockcraft issue](https://github.com/canonical/rockcraft/issues/242).